portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -117,40 +115,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty("comment", comment);
     commentEntity.setProperty("time_millis", System.currentTimeMillis());    
     datastore.put(commentEntity);
-    updateWordCount(comment);
+    new WordCountUpdater().updateWordCount(comment, '+');
```

Comment:
Store a class instance of WordCountUpdater instead of relying on the fact that it uses the same underlying data store.